languages/csharp/reference/exercise-concepts/protein-translation.md

```diff
@@ -0,0 +1,18 @@
+# Concepts of protein-translation
+
+- using keyword: for importing libraries
+- access modifiers: in order to specify public vs private functions
+- static keyword: in order to class/methods accessible without having to instantiate the class
+- arrays: for the return value from the main function
+- functions: used as the main entry point for the exercise
+- function arguments: input strand is passed as an arguments
+- var keyword: in order to prevent verbose declarations `List<string> results = new List<string>()`
+- assignment: setting a variable to a specific value
+- new keyword: used to instantiate a class
+- List<T>: temporarily hold the results before converting them to an array (Add and ToArray methods)
+- for loop: iterate through the strands
+- int type: used as a counter in the loop

```

Comment:
Maybe do:

```suggestion
- integers: `int` type used as a counter in the loop 
```

To better indicate that the _concept_ used is integers, which C# implements through an `int`.